libyui-ncurses/pkgconfig/libyui-ncurses.pc.in

```diff
@@ -0,0 +1,23 @@
+#
+# Pkg-Config file for libyui-ncurses
+#
+# Generated by CMake from libyui-ncurses.pc.in
+#
+
+prefix=@CMAKE_INSTALL_PREFIX@
+exec_prefix=${prefix}
+
+libdir=${exec_prefix}/@CMAKE_INSTALL_LIBDIR@
+includedir=${exec_prefix}/include/yui/ncurses
+plugindir=${exec_prefix}/@CMAKE_INSTALL_LIBDIR@/yui
+
+soversion_major=@SONAME_MAJOR@
+soversion_minor=@SONAME_MINOR@
+soversion_patch=@SONAME_PATCH@
+soversion=@SONAME@
+
+Name: libyui-ncurses
+Version: @VERSION@
+Description: libyui-ncurses - NCurses plug-in for libyui
+Libs: -L${plugindir} -lyui-ncurses
```

Comment:
ops I missed plugindir is been used here, sorry

Reply:
No problem.

Thanks for reviewing!